From d3c23e8c545331a4aef2fdbe0a64924e55be1abc Mon Sep 17 00:00:00 2001
From: Andriy Petlovanyy <andriy.petlovanyy@noemail.com>
Date: Fri, 14 Sep 2018 09:43:10 +0000
Subject: [PATCH] Copied postinst to debian package

---
 debian/postinst | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100755 debian/postinst

diff --git a/debian/postinst b/debian/postinst
new file mode 100755
index 0000000..56eb647
--- /dev/null
+++ b/debian/postinst
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+ROS_DISTRO=melodic
+EXEC_NAME=ethercat_grant
+case "$1" in
+ configure)
+   chmod +s /opt/ros/$ROS_DISTRO/bin/$EXEC_NAME
+   ;;
+ *)
+       echo "postinst called with unknown argument \`$1'" >&2
+       exit 0
+   ;;
+esac
+#DEBHELPER#
-- 
2.7.4

